genesis.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can you delete this and the pre-seal file?

Reply:
and maybe also add `genesis.json` to the gitignore, since it is the default file path (right?)

Reply:
oops, it's slipped through, didn't intend to add them. Removed now. Added `genesis.json` to gitignore.